target/linux/rockchip/image/armv8.mk

```diff
@@ -4,6 +4,17 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+
+define Device/friendlyarm_nanopi-r2s
+  DEVICE_VENDOR := FriendlyARM
+  DEVICE_MODEL := NanoPi R2S
+  SOC := rk3328
+  UBOOT_DEVICE_NAME := nanopi-r2s-rk3328
+  IMAGE/sysupgrade.img.gz := boot-common | boot-script nanopi-r2s | pine64-img | gzip | append-metadata
+  DEVICE_PACKAGES := kmod-usb-net kmod-usb-net-rtl8152
```

Comment:
kmod-usb-net is selected by kmod-usb-net-rtl8152, so could be removed

Reply:
It is not, it only has the dependency flagged without autoselection.

Reply:
Oops, my eye was moving the plus to the wrong side of the equal sign. Sorry.

I will think about whether that should be made a +kmod-usb-net in AddDepends/usb-net separately ...